src/GraphQL/Execution/ExecutionHelper.cs

```diff
@@ -65,9 +65,7 @@ public static FieldType GetFieldDefinition(Document document, ISchema schema, IO
 
             if (parentType == null)
             {
-                var error = new ExecutionError($"Schema is not configured correctly to fetch {field.Name}.  Are you missing a root type?");
-                error.AddLocation(field, document);
-                throw error;
+                throw new ArgumentNullException(nameof(parentType), $"Schema is not configured correctly to fetch {field.Name}.  Are you missing a root type?");
```

Comment:
`parentType` should never be null unless you're querying for a introspection type.  After this change, `document` is not needed and was removed from the method signature